File: hw_06/Java/java_oopwork06/src/clothes/Shirt.java
public class Shirt {
    private Color color;
    private Size size;
    private double price;

    public Shirt() {
        this(Color.Black, Size.L, 199.00);
    }

    public Shirt(Color color, Size size, double price) {
        this.color = color;
        this.size = size;
        this.price = price;
    }

    public void shirtInfo() {
        System.out.println("Shirt: [ Color: " + color + ", Size: " + size + ", Price: " + price + "]");
    }

    public Color getColor() {
        return color;
    }

    public void setColor(Color color) {
        this.color = color;
    }

    public Size getSize() {
        return size;
    }

    public void setSize(Size size) {
        this.size = size;
    }

    public double getPrice() {
        return price;
    }

    public void setPrice(double price) {
        this.price = price;
    }
}

File: hw_06/CS/cs_oopwork06/cs_oopwork06/clothes/Shirt.cs

namespace cs_oopwork06.clothes
{
    public class Shirt
    {
        private Color color;
        private Size size;
        private double price;

        public Color GetColor()
        {
            return color;
        }

        public void SetColor(Color color)
        {
            this.color = color;
        }

        public Size GetSize()
        {
            return size;
        }

        public void SetSize(Size size)
        {
            this.size = size;
        }

        public double GetPrice()
        {
            return price;
        }

        public void SetPrice(double price)
        {
            this.price = price;
        }

        public Shirt(Color color, Size size, double price)
        {
            this.color = color;
            this.size = size;
            this.price = price;
        }

        public Shirt(): this(Color.Black, Size.L, 199.00){}

        public void ShirtInfo()
        {
            System.Console.WriteLine($"Shirt: Color {color}, Size {size}, Price {price:N2}");
        }
    }
}